development/plans/eclipse_project_plan_4_28.xml

```diff
@@ -0,0 +1,424 @@
+				OpenJDK 17.0.3 (LTS)<br/>
+			</td>
+			<td rowspan="2">GTK 3</td>
+		</tr>
+		<tr class="c0">
+			<td rowspan="1">Power 64-bit LE</td>
+			<td rowspan="1">
+				OpenJDK 11.0.15 (LTS)
+			</td>
+		</tr>
+
+		<!-- ************ Ubuntu ************** -->
+		<tr class="c1">
+			<td rowspan="1">Ubuntu Long Term Support</td>
+		    <td rowspan="1">22.04</td>
+			<td rowspan="1">
+				x86 64-bit<br/>
+				aarch64<br/>
+			</td>
+			<td rowspan="1">
+				OpenJDK 17.0.3 (LTS)<br/>
+			</td>
+			<td rowspan="1">GTK 3</td>
+		</tr>
+		
+		<!-- ************ Mac ************** -->
+		<tr class="c1">
+			<td rowspan="2">Apple macOS</td>
+			<td rowspan="1">
+				12<br/>
+				13
+			</td>
+			
+			<td rowspan="1">x86 64-bit</td>
+			
+			<td rowspan="1">
+				OpenJDK 17.0.3 (LTS)<br/>
+				Oracle Java 17.0.3 (LTS)
+			</td>
+			
+			<td rowspan="2">Cocoa</td>
+		</tr>
+ 	    <tr class="c0">
+			<td rowspan="1">
+				12<br/>
+				13
+			</td>
+			<td rowspan="1">
+				M1 (arm64)
+			</td>
+            <td rowspan="1">OpenJDK 17.0.3 (LTS)</td>
+   		</tr>
+	</table>
+ </center>
+
+<p>As stated above, <i>we expect that Eclipse works fine on other current
+  Java VM and OS versions but we cannot flag these as reference platforms without
+  significant community support for testing them.</i></p>
+<p>With respect to GTK 3 versions: the <i>required</i> version of GTK 3 which Eclipse SDK 4.28
+will run on is 3.22 or newer.
+</p>
+
+<p>With respect to GTK 3 themes: Adwaita theme is guaranteed to work. Eclipse SDK will run with other GTK 3 themes,
+however we cannot flag these as reference versions without significant community support for testing and/or development
+of fixes. Bugs that are reproducible only with themes other than Adwaita will be given a lower priority (or may not be fixed at all),
+compared to bugs which are reproducible on the target environments listed above.
+</p>
+</div>
+
+<p:internationalization>
+<p>The Eclipse SDK is designed as the basis for internationalized products. The
+  user interface elements provided by the Eclipse SDK components, including dialogs
+  and error messages, are externalized. The English strings are provided as the
+  default resource bundles.</p>
+<p>Latin-1, DBCS, and BiDi locales are supported by the Eclipse SDK on all reference platforms.</p>
+<p>The Eclipse SDK supports GB 18030 (level 1), the Chinese code page standard,
+  on Windows, Linux and the Macintosh.</p>
+</p:internationalization>
+
+</p:target_environments>
+ 
+<!-- ============================================== -->
+
+<p:compatibility_with_previous_releases>
+<div>
+
+<h3>Compatibility of Release 4.28 with 4.27</h3>
+<p>Eclipse 4.28 will be compatible with Eclipse 4.27</p>
+
+<p><strong>API Contract Compatibility:</strong> Eclipse SDK 4.28 will be upwards
+  contract-compatible with Eclipse SDK 4.27 except in those areas noted in the
+  <a href="https://www.eclipse.org/eclipse/development/porting/eclipse_4_27_porting_guide.html" target="_top">
```

Comment:
Links should also be updated to point to 4_28 for example eclipse_4_28_porting_guide.html (though the page is not available yet). Same for the other 4_27 links.